### PR TITLE
Update to vanilla 4.0 and adjust some styles

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- updated vanilla to 4.0
- adjusted styles of several headings

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the style of pages, especially headings
    - instance/profile overview categories visual changed slightly
    - create and add certificate visual changed slightly

### new layout:
Screenshot from 2023-07-07 12-40-19

### old layout:
Screenshot from 2023-07-07 12-40-28

### CODE REVIEW

File: src/pages/instances/InstanceOverview.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ const InstanceOverview: FC<Props> = ({ instance }) => {
       />
       <Row className="general">
         <Col size={3}>
-          <h2 className="p-heading--4">General</h2>
+          <h2 className="p-heading--5">General</h2>
```
Comment (lorumic) [inline src/pages/instances/InstanceOverview.tsx:47]: I think this requires design input. I don't see a valid reason to suddenly change this style.

**Review by lorumic — COMMENTED** [review]
The heading updates seem a bit arbitrary to me. Let's see if we can get a proper design review for this first.

File: src/pages/instances/InstanceOverview.tsx (hunk 1 above)
Comment (edlerd) [reply src/pages/instances/InstanceOverview.tsx:47]: @piperdeck can you have a look at this? Is the change to new layout (bolded section title in the same font size as the rest of the content) from the old layout (bigger font for the section, not bold) acceptable?

**Review by lorumic — APPROVED** [review]
Nothing to object, code-wise.

**Pull request merged**